lib/ruby_indexer/lib/ruby_indexer/index/entry.rb

```diff
@@ -0,0 +1,207 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyIndexer
+  class Index
```

Comment:
How about we get rid of the `Index` namespace while we're at it? We don't get much out of it.

Reply:
👍 updated